at_client/lib/src/service/sync_service_impl.dart

```diff
@@ -339,7 +339,6 @@ class SyncServiceImpl implements SyncService, AtSignChangeListener {
           } on Exception catch (e) {
             _logger.severe(
                 'exception while updating commit entry for entry:$entry ${e.toString()}');
-            return keyInfoList;
```

Comment:
need to also remove the return statement at line 348 (now line 347)

Reply:
done